docs/blog/2019-06-25-profitable-agency-wordpress/index.md

```diff
@@ -62,15 +62,15 @@ However, if your job is to build custom WordPress solutions for large sites, the
 
 ## Hosting for Gatsby sites starts at \$0
 
-Last year I built a WordPress website for an event. I used a few popular membership and ecommerce plugins to get the job done. The price of hosting and plugins were all obscured from the client. I was paid a fee, and then I turned around and paid for hosting and plugins, and pocketed the margin.
+Last year I built a WordPress website for an event. I used a few popular membership and E-commerce plugins to get the job done. The price of hosting and plugins were all obscured from the client. I was paid a fee, and then I turned around and paid for hosting and plugins, and pocketed the margin.
 
 I even collected an affiliate fee from the server host. Again, if site maintenance is your bread-and-butter, then you know this model. Every nickel counts.
 
-After the event, my client no longer needed the membership and ecommerce features. And they weren't enthusiastic about paying me for ongoing support. So, instead of haggling over a price, or handing over the project and losing their business forever, I turned the WordPress project into a static site that I host on Netlify.
+After the event, my client no longer needed the membership and E-commerce features. And they weren't enthusiastic about paying me for ongoing support. So, instead of haggling over a price, or handing over the project and losing their business forever, I turned the WordPress project into a static site that I host on Netlify.
```

Comment:
```suggestion
After the event, my client no longer needed the membership and e-commerce features. And they weren't enthusiastic about paying me for ongoing support. So, instead of haggling over a price, or handing over the project and losing their business forever, I turned the WordPress project into a static site that I host on Netlify.
```